frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -445,41 +446,41 @@ export function ObjectDetailHeader({
           {objectId !== null && <ObjectIdLabel> {objectId}</ObjectIdLabel>}
         </h2>
       </div>
-      {showActions && (
-        <div className="flex align-center">
-          <div className="flex p2">
-            {!!canZoom && (
-              <>
-                <Button
-                  data-testid="view-previous-object-detail"
-                  onlyIcon
-                  borderless
-                  className="mr1"
-                  disabled={!canZoomPreviousRow}
-                  onClick={viewPreviousObjectDetail}
-                  icon="chevronup"
-                />
-                <Button
-                  data-testid="view-next-object-detail"
-                  onlyIcon
-                  borderless
-                  disabled={!canZoomNextRow}
-                  onClick={viewNextObjectDetail}
-                  icon="chevrondown"
-                />
-              </>
-            )}
-            <CloseButton>
+
+      {showControls && (
+        <Flex align="center" p="1rem">
```

Comment:
I think in general if we can it'd be ideal to use an value from the scale like `2` or something here rather than `1rem` so there's an association with the theme. Let me know if I'm missing some context though.

Reply:
@kdoh Did you mean something like `<Flex align="center" p="md">`?
If not, can you please provide a code snippet?

Reply:
Yup that's the idea, just using a value from the spacing scale.

Reply:
Done: 3d58d697b8b1d2b5ac0f78f5889d28dbc57add45

Previously the horizontal spacing between these buttons was `0.5rem`.
The `gap` prop of `Flex` gives:
- `0.625rem` for `xs`
- `0.75rem` for `sm`
- `1rem` for `m`

I went with `xs` as it's the closest to `0.5rem`.

Reply:
I think these values were never approved, they are just mantine defaults, I don't think we could/should use them right now

Reply:
> I think these values were never approved, they are just mantine defaults, I don't think we could/should use them right now

Thanks! Updated in another branch: 28d03593aca9b1db9cd2fb4124f7d54b6e7f56af